frontend/src/pages/orgs/[orgId]/events/[eventId].tsx

```diff
@@ -126,141 +131,244 @@ const EventAdminPage: NextPage = () => {
+                    <Typography align="center" variant="body1">
+                      Ingen påmeldte
+                    </Typography>
+                  )}
+                </Box>
+              </AccordionDetails>
+            </Accordion>
+          ) : null}
+          {data?.event?.attendable ? (
+            <Accordion>
+              <AccordionSummary aria-controls="panel3a-content" id="panel3a-header">
+                <Box ml={5} my={2}>
+                  <CardHeader titleTypographyProps={{ variant: "h4" }} title="Venteliste" />
+                </Box>
+              </AccordionSummary>
+              <AccordionDetails>
+                <Box sx={{ mb: 2, mx: 5, width: "-webkit-fill-available" }}>
+                  {data?.event?.usersOnWaitingList?.length !== 0 ? (
+                    <TableContainer style={{ maxHeight: 600 }}>
+                      <Table>
+                        <TableHead>
+                          <TableRow>
+                            {signUpFields.map((field) => (
+                              <TableCell key={`user-header-${field.header}`}>{field.header}</TableCell>
                             ))}
-                          </TableBody>
-                        </Table>
-                      </TableContainer>
-                    ) : (
-                      <Typography align="center" variant="body1">
-                        Ingen påmeldte
-                      </Typography>
-                    )}
-                  </CardContent>
-                </Card>
-              </Grid>
-              <Grid item xs>
-                <Card variant="outlined">
-                  <CardHeader title="Venteliste" />
-                  <CardContent>
-                    {data?.event?.usersOnWaitingList?.length !== 0 ? (
-                      <TableContainer style={{ maxHeight: 600 }}>
-                        <Table>
-                          <TableHead>
-                            <TableRow>
+                          </TableRow>
+                        </TableHead>
+                        <TableBody>
+                          {data?.event?.usersOnWaitingList?.map((signUp: SignUp) => (
+                            <TableRow key={`user-row-${signUp.user.id}`}>
                               {signUpFields.map((field) => (
-                                <TableCell key={`user-header-${field.header}`}>{field.header}</TableCell>
+                                <TableCell key={`user-${signUp.user.id}-cell--${field.field}`}>
+                                  {field.header === "Navn"
+                                    ? `${signUp.user.firstName} ${signUp.user.lastName}`
+                                    : field.header === "Mobilnummer"
+                                    ? signUp.userPhoneNumber.slice(3)
+                                    : signUp[field.field]
+                                    ? signUp[field.field]
+                                    : "━"}
+                                </TableCell>
```

Comment:
Either make a component out of these cells, or refactor the logic into a helper function.